tests/Resources/RHOSi.resource

```diff
@@ -40,3 +41,8 @@ Protect Sensitive Variables In Keywords
     [Documentation]    Register keywords which use sensitive data as "Protected"
     ...                to turn their log level to NONE using RobotLogListener by Robocorp
     Register Protected Keywords    names=@{PROTECTED_KEYWORDS}
+
+Initialize Global Variables
+    [Documentation]   Sets global variables
+    ${RHODS_VERSION}=    Get RHODS Version
```

Comment:
force-reload=True   (see above comment)

Reply:
I have used force fetch and by default it will be false